packages/gatsby-theme-docs/src/layouts/release-notes-detail.js

```diff
@@ -50,24 +50,23 @@ const LayoutReleaseNotesDetail = (props) => {
                     font-size: ${designSystem.typography.fontSizes.small};
                   `}
                 >
-                  {'Back to all releases'}
+                  {`All ${siteData.siteMetadata.title} Releases`}
```

Comment:
Silly UX question: does the link need to have the site name in the text? It gets kind of redundant as it's visible above and on the left side of the link 🤔

<img width="566" alt="image" src="https://user-images.githubusercontent.com/1110551/79731803-15a4b100-82f3-11ea-8b84-9a564c349cfc.png">

Reply:
we also just can use "< Back"

Reply:
@Shecki @emmenko @nkuehn please let me know what the final decision of the text of the "back link" should be.

Reply:
FYI: It can also be a follow up, it doesn't need to block this PR

Reply:
I think it should say "All Release Notes".